NFIQ2/NFIQ2Algorithm/src/features/FingerJetFXFeature.cpp

```diff
@@ -247,38 +169,13 @@ FingerJetFXFeature::computeFeatureData(
 		res_min_cnt.featureData = fd_min_cnt;
 		featureDataList.push_back(res_min_cnt);
 
-		if (m_bOutputSpeed) {
-			NFIQ::QualityFeatureSpeed speed;
-			speed.featureIDGroup =
-			    FingerJetFXFeature::speedFeatureIDGroup;
-			speed.featureIDs.push_back("FingerJetFX_MinutiaeCount");
-			speed.featureIDs.push_back(
-			    "FingerJetFX_MinCount_COMMinRect200x200");
-			speed.featureSpeed = timer.endTimerAndGetElapsedTime();
-			m_lSpeedValues.push_back(speed);
-		}
```

Comment:
Still needed?

Reply:
Yes. Re-adding. This was part of a check that checked if the min count was equal to 0.